api/migrations/38-add-friend-user.sql

```diff
@@ -0,0 +1,37 @@
+--Check if current user is allowed to access past ctfs
+CREATE FUNCTION ctfnote_private.is_friend ()
+  RETURNS boolean
+  AS $$
+  SELECT
+    ctfnote_private.current_role () = 'user_friend'::ctfnote.role
+    OR ctfnote_private.is_member ();
```

Comment:
Shouldn't we redefine `is_guest` to check for `is_friend` instead of `is_member` ?

Reply:
You are totally right. Good catch!